crates/bevy_app/src/app.rs

```diff
@@ -14,15 +14,14 @@ use std::fmt::Debug;
 
 #[cfg(feature = "trace")]
 use bevy_utils::tracing::info_span;
-
 bevy_utils::define_label!(AppLabel);
 
 #[allow(clippy::needless_doctest_main)]
 /// Containers of app logic and data
 ///
 /// Bundles together the necessary elements, like [`World`] and [`Schedule`], to create
 /// an ECS-based application. It also stores a pointer to a
-/// [runner function](App::set_runner), which by default executes the App schedule
+/// [runner function](Self::set_runner), which by default executes the App schedule
```

Comment:
I think that the information about the default settings belongs elsewhere: it isn't going to be relevant to 90% of the users reading these docs.

Instead, I'd steal very briefly from below, and explain that "the runner is responsible for managing the application's event loop and applying the [`Schedule`] to the ['World'] to drive application logic."